pkg/apis/kubermatic/v1/settings.go

```diff
@@ -249,6 +252,14 @@ type StaticLabel struct {
 	Protected bool     `json:"protected"`
 }
 
+type Announcement struct {
+	Message   string `json:"message"`
+	Id        string `json:"id"`
+	Status    string `json:"status"`
+	CreatedAt string `json:"createdAt"`
+	Expires   string `json:"expires,omitempty"`
```

Comment:
Now that you wrote the feature is the perfect time to document each of these fields.

The Go field should be `ID`, not `Id`.

Is the ID meant to be unique? If so, how do we ensure this? Is the order of announcements important? Otherwise maybe a map would be a better datastructure?

Both date fields should be `time.Time`, and Expires can be a `*time.Time` even.